python/ray/dataframe/dataframe.py

```diff
@@ -776,9 +776,20 @@ def dropna(self, axis=0, how='any', thresh=None, subset=None,
             DataFrame with the dropna applied.
         """
         if is_list_like(axis):
-            raise NotImplementedError(
-                "To contribute to Pandas on Ray, please visit "
-                "github.com/ray-project/ray.")
+            axis = set([pd.DataFrame()._get_axis_number(ax) for ax in axis])
+            result = self
```

Comment:
unnecessary

Reply:
I think it is necessary in this case b/c the for loop makes reference to `result`. this is consistent with the pandas source code.